mobify/progressive-web-scaffold
# Cart edit button

This PR fixes the cart item edit button so it correctly navigates to the PDP

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1110
 **Linked PRs**:n/a

## Changes
- Added href to cart item edit button
- Added PDP route for configure item URL

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart
- Click "edit" for the item in you cart
- You should be taken to the PDP for the item
